tests/parker.js

```diff
@@ -12,7 +12,7 @@ function generateMetrics(file, metricsArray) {
     {
       name: 'Stylesheet size',
       benchmark: 150000,
-      threshold: 200000,
+      threshold: 250000,
```

Comment:
If you're going to do this, I'd urge you to add a card and plan some time to refactor Vanilla so it's back below 200000. Typically we shouldn't change the tests so that the code passes, but rather change the code so that the test passes, otherwise the tests serve no purpose. i.e. Why stop at 250000 if whenever the threshold is reached its simply raised?

Reply:
The problem is, these are tiny commits, so there's nothing here to refactor, and all open issues would be blocked if we stick to the current limit. We need an issue to see if it is organic growth, or something that created an unreasonable amount of output. I've created an issue [here](https://github.com/canonical-web-and-design/vanilla-framework/issues/2657):

Reply:
Nice! Yep, wasn't suggesting this should block merging work, just disagreed with increasing the threshold without a plan to investigate or improve.

Reply:
Did you not need this in the end?

Reply:
We need it until we optimise or drop newly added icons that are mainly causing the file size increase. I've made an [issue](https://github.com/canonical-web-and-design/vanilla-framework/issues/2657) for it.